src/utilities/show.jl

```diff
@@ -13,13 +13,13 @@ end
 # sign: NoSign()
 # vexity: AffineVexity()
 function show(io::IO, x::Variable)
-  print(io, """Variable of
-    size: ($(x.size[1]), $(x.size[2]))
-    sign: $(x.sign)
-    vexity: $(x.vexity)""")
-  if x.value != nothing
-    print(io, "\nvalue: $(x.value)")
-  end
+    print(io, """Variable of
+        size: ($(x.size[1]), $(x.size[2]))
+        sign: $(x.sign)
+        vexity: $(x.vexity)""")
```

Comment:
Indentation of the multiline prints are off here

Reply:
In what sense? This just pushes them out another 2 spaces, which should have no effect on how they're printed because of the placement of the closing triple quote.

Reply:
You are correct in that the prints display the same way the did before. What I was referring to was that I think we may want the multiline string to start at the beginning of the `(`. I don't use this style convention myself but I think this is probably the most consistent:
```julia
print(io, """Variable of
      size: ($(x.size[1]), $(x.size[2]))
      sign: $(x.sign)
      vexity: $(x.vexity)""")
```